src/dlgTriggerEditor.cpp

```diff
@@ -3507,13 +3511,11 @@ void dlgTriggerEditor::addVar(bool isFolder)
     }
     vu->addTempVar(pNewItem, newVar);
     pNewItem->setFlags(pNewItem->flags() & ~(Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled));
-// The following test is pointless - we will already have seg. faulted if pNewItem is a nullptr...!
```

Comment:
I commented this test out for the stated reason - but can `pNewItem` ever **be** a `nullptr`?

Reply:
I'm not sure, but I don't see the point in having commented out code when clearly it's effective already :)